participants/Pufferfish Cloud Storage Drive.md

```diff
@@ -0,0 +1,53 @@
+# Pufferfish Cloud Storage Drive
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+Project Name: Pufferfish Cloud Storage Drive, Our slack Handle: U017LEUM8HF
+Username: 2966696199

```

Comment:
could you please add your Slack username handle here please, should be in the `@xxx` format.

Reply:
@PufferDriver , Thanks.